docs/admin-guides/managing-optimizers.md

```diff
@@ -123,15 +123,28 @@ Currently, only pptimizer scaled through the dashboard can be released on dashbo
 You can submit optimizer in your own Flink task development platform or local Flink environment with the following configuration. The main parameters include:
 
 ```shell
-flink run -m yarn-cluster  -ytm {EXECUTOR_TASKMANAGER_MEMORY} -yjm {EXECUTOR_JOBMANAGER_MEMORY}  -c com.netease.arctic.optimizer.flink.FlinkOptimizer  {ARCTIC_HOME}/plugin/optimize/OptimizeJob.jar -a {AMS_THRIFT_SERVER_URL} -g {OPTIMIZE_GROUP_NAME} -p {EXECUTOR_PARALLELISM} -m {EXECUTOR_MEMORY}  --hb 60000
+./bin/flink run-application -t yarn-application \
+ -Djobmanager.memory.process.size=1024m \
+ -Dtaskmanager.memory.process.size=2048m \
+ -c com.netease.arctic.optimizer.flink.FlinkOptimizer \
+ ${ARCTIC_HOME}/plugin/optimize/OptimizeJob.jar \
+ -a 127.0.0.1:1261 \
+ -g flinkGroup \
+ -p 1 \
+ -eds \
+ -dsp /tmp \
+ -msz 512 \
+ -id 9c12edca-b0bf-434d-a954-dd8aab88a284
```

Comment:
External optimizer do not need this `id` parameter.

Reply:
Done.